app/views/admin/enterprise_roles/_data.html.haml

```diff
@@ -1,4 +1,3 @@
-= admin_inject_enterprise_roles(@enterprise_roles)
-= admin_inject_users(@users)
```

Comment:
It looks like these two helpers are now unused and can be deleted.

Reply:
updated, thanks